lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2483,49 +2484,45 @@ def quiver(self, *args,
                length=1, arrow_length_ratio=.3, pivot='tail', normalize=False,
                **kwargs):
         """
-        Plot a 3D field of arrows.
-
-        call signatures::
-
-            quiver(X, Y, Z, U, V, W, **kwargs)
-
-        Arguments:
+        ax.quiver(X, Y, Z, U, V, W, /, length=1, arrow_length_ratio=.3, pivot='tail', normalize=False, **kwargs)
 
-            *X*, *Y*, *Z*:
-                The x, y and z coordinates of the arrow locations (default is
-                tail of arrow; see *pivot* kwarg)
-
-            *U*, *V*, *W*:
-                The x, y and z components of the arrow vectors
+        Plot a 3D field of arrows.
 
         The arguments could be array-like or scalars, so long as they
         they can be broadcast together. The arguments can also be
         masked arrays. If an element in any of argument is masked, then
         that corresponding quiver element will not be plotted.
 
-        Keyword arguments:
+        Parameters
+        ----------
+        X, Y, Z : array-like
+            The x, y and z coordinates of the arrow locations (default is
+            tail of arrow; see *pivot* kwarg)
```

Comment:
I haven't and don't intend to touch any of these descriptions - my goal was only to switch to using `Parameters`

Reply:
:+1: